student.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+
+class Student
+  attr_accesor :classroom
+
+  include person
```

Comment:
- Please, for readability's sake, kindly make the Student class inherit from the parent class Person.
`class Student < Person`